docs/rst/manual/reference/configuration/magic_strings.rst

```diff
@@ -0,0 +1,259 @@
+
+Magic strings
+=============
+
+ERT exposes some magic strings that can be used in your configuration file.
+These strings will at runtime be replaced with corresponding values, and
+can be useful if you need to use information such as file paths or experiment
+information such as realization or iteration number in your workflows or jobs.
+
+Example:
+::
+
+   YOUR_WORKFLOW_JOB <RUNPATH_FILE> <CONFIG_PATH>
+
+Will be replaced with something similar to the following at runtime
+
+::
+
+   YOUR_WORKFLOW_JOB /path/to/the/runpath/file/.ert_runpath_list /path/to/config/
+
+In the following we separate between :ref:`experiment constants <ms_exp_constants>`
+and :ref:`experiment variables <ms_exp_variables>`. Experiment are magic strings that
+do not change based on the iteration or realization number, while experiment variables
+will typically be replaced with different values depending on the current iteration or
+realization index.
+
+**It should also be noted that experiment variables can only be used with forward model jobs
+and not with workflows.**
+
+
+List of keywords
+----------------
+
+
+* Experiment Constants
+
+   * :ref:`RUNPATH_FILE <ms_runpath_file>`
+   * :ref:`CONFIG_PATH <ms_config_path>`
+   * :ref:`CONFIG_FILE <ms_config_file>`
+   * :ref:`CONFIG_FILE_BASE <ms_config_file_base>`
+   * :ref:`CWD <ms_cwd>`
+
+* Experiment Variables
+
+   * :ref:`IENS <ms_iens>`
+   * :ref:`ITER <ms_iter>`
+   * :ref:`ECLBASE/ECL_BASE <ms_eclbase>`
+   * :ref:`GEOID <ms_geoid>`
+   * :ref:`RUNPATH <ms_runpath>`
+   * :ref:`NUM_CPU <ms_num_cpu>`
```

Comment:
This needs to be checked :) Whether `NUM_CPU` is actually substituted that is...

Reply:
It is :slightly_smiling_face: When you run the poly case it is replaced with 1, but there seems to be code for fetching the number of CPUs from the eclipse data file. So my plan was to check if it will actually fetch it from a DATA file also.